python/mxnet/gluon/block.py

```diff
@@ -1335,12 +1337,21 @@ def export(self, path, epoch=0, remove_amp_cast=True):
         arg_names = set(sym.list_arguments())
         aux_names = set(sym.list_auxiliary_states())
         arg_dict = {}
-        for param in self.collect_params().values():
-            if param.name in arg_names:
-                arg_dict['arg:%s'%param.name] = param._reduce()
-            else:
-                assert param.name in aux_names
-                arg_dict['aux:%s'%param.name] = param._reduce()
+        if self._cached_op_args:
```

Comment:
When would self._cached_op_args not be present?

Reply:
I was missing that export requires hybridization, fixing it now